spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+RSpec.describe 'User show page', type: :feature do
+  let(:user) { create(:user) }
+  let!(:posts) do
+    create_list(:post, 3, author: user)
+  end
+  before do
+    visit user_path(user)
+  end
+  it 'shows the user profile with their posts' do
+    expect(page).to have_content(user.name)
+    expect(page).to have_content("Number of posts: #{user.posts.size}")
+    user.recent_posts.each do |post|
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text)
+      expect(page).to have_content("Comments: #{post.comments.size}")
+      expect(page).to have_content("Likes: #{post.likes_counter}")
+    end
+  end
```

Comment:
- [ ] Please kindly add the missing tests as highlighted in the screenshot below. 👇 
 
<img width="453" alt="Untitled" src="https://github.com/shm04/blogApp/assets/103386183/1101911a-d773-49ce-8b30-633ac2503445">